src/projects/apps/auth/authorization.md

```diff
@@ -0,0 +1,59 @@
+## Authorization
+
+### Users, Groups and Roles
+
+The authorization framework uses the concept of Groups and Roles to manage access to resources and content in your app.
+
+A **Role** is a named permission that can be placed on resources or content to restrict the level of access e.g. `Manager`, `Contributor`, `Viewer`.
+
+A **Group** is a collection of one or more roles and a user is assigned to one or more groups. Some examples of groups might include `Line Managers`, `Senior Engineer`, `Junior Engineer`.
```

Comment:
group names should probable all be plural

Reply:
But then I have to redo my image 😆